### PR TITLE
Ui tox update

### PR DESCRIPTION
**Pull request by rkhozinov**

### CODE REVIEW

File: servers/partner-ci/10.0/zabbix-nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+- runner-condition
+- multijob:
+    name: 'Smoke/BVT test for dvs{fuel_release}.{plugin_name}.{plugin_version}'
```
Comment (naumvd95) [inline servers/partner-ci/10.0/zabbix-nightly.yaml:3]: this is very strange
zabbix cases are already presented in another PR
need to discuss that

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/10.0/zabbix-nightly.yaml (hunk 1 above)
Comment (rkhozinov) [reply servers/partner-ci/10.0/zabbix-nightly.yaml:3]: ok. I'll fix that

File: servers/partner-ci/10.0/zabbix.yaml [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+      - uninstall_zabbix_monitoring
+
+    jobs:
+      - '{fuel_release}.{plugin_name}.build':
+          gerrit_repo:   '{zbx_emc_repo}'
+          plugin_name:   '{zbx_emc_repo}'
+          gerrit_branch: '{zbx_emc_branch}'
+
+      - '{fuel_release}.{plugin_name}.build':
+          gerrit_repo:   '{zbx_extreme_net_repo}'
```
Comment (naumvd95) [inline servers/partner-ci/10.0/zabbix.yaml:108]: seems we need to rename builds differently

File: servers/partner-ci/10.0/zabbix.yaml [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+          gerrit_repo:   '{zbx_ext_repo}'
+          plugin_name:   '{zbx_ext_repo}'
+          gerrit_branch: '{zbx_ext_branch}'
+
+      - '{nightly-name}':
+          multijob_build_pkg: true
+
+#     - '{custom-name}.qa':
+#         publish_results: false
+
```
Comment (naumvd95) [inline servers/partner-ci/10.0/zabbix.yaml:127]: why u comment that, but define custom-job-templates?

File: servers/partner-ci/10.0/zabbix.yaml [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+      - string:
+          name: PUPPETLINT_IGNORE
+          default: '{puppetlint_ignore}'
+      - bool:
+          name: DEBUG
+          default: '{build_debug}'
+    properties:
+      - test-property
+    wrappers:
+      - ansicolor:
```
Comment (naumvd95) [inline servers/partner-ci/10.0/zabbix.yaml:355]: we are already  have xterm case in "runner wrapper", which u defined above

File: servers/partner-ci/10.0/zabbix.yaml [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+    project-type: multijob
+    disabled: '{obj:nightly_disabled}'
+    node: runner
+    <<: *runner-parameters
+    scm: []
+    triggers:
+      - timed: '{nightly_timer}'
+    properties:
+      - runner-property:
+          build-name: '{build-name}'
```
Comment (naumvd95) [inline servers/partner-ci/10.0/zabbix.yaml:306]: seems we need to replace it with ui-runner-property
and definitely we need to rename ui-runner-property

File: servers/partner-ci/builders/build-plugin-zabbix.sh [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+         --no-arrow_alignment-check
+
+fpb --check  ./
+fpb --build  ./
+
+if [[ "${DEBUG}" == "true" ]]; then
+  fpb --debug --build  ./
+else
+  fpb --build  ./
+fi
```
Comment (naumvd95) [inline servers/partner-ci/builders/build-plugin-zabbix.sh:29]: useless checker
as i see above, "+fpb --build  ./" is always performed

File: servers/partner-ci/builders/zabbix.test.sh [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+last_snapshot=$(dos.py snapshot-list ${ENV_NAME} | tail -1 | cut -d' ' -f1)
+dos.py revert-resume ${ENV_NAME} "${last_snapshot}"
+
+
+cd ${WORKSPACE}/docker/
+pip install --upgrade docker-compose
+ln -s ${WORKSPACE}/fuel-ui fuel-ui
+docker-compose down -v
+docker-compose up --remove-orphans --build --abort-on-container-exit
+
```
Comment (naumvd95) [inline servers/partner-ci/builders/zabbix.test.sh:30]: need to remove ui-case

File: tox.ini [hunk 8]
```diff
@@ -25,5 +25,9 @@ commands =
     9.{x,1,0},{8,7}.0: {env:CMD}/{envname}/vcenter.yaml:{env:COMMON}         -o {env:OUTPUT}
     9.{x,1,0}:         {env:CMD}/{envname}/nsxt.yaml:{env:COMMON}            -o {env:OUTPUT}
     9.x:               {env:CMD}/{envname}/upgrade.yaml:{env:COMMON}         -o {env:OUTPUT}
-    9ui:               {env:CMD}/9.x/ui.yaml:{env:COMMON}                    -o {env:OUTPUT}
-    10ui:              {env:CMD}/10.0/ui.yaml:{env:COMMON}                   -o {env:OUTPUT}
\ No newline at end of file
+    9.x,10.0:          {env:CMD}/{envname}/ui.yaml:{env:COMMON}              -o {env:OUTPUT}
+    10.0:              {env:CMD}/{envname}/fuel.yaml:{env:COMMON}            -o {env:OUTPUT}
+    10.0:              {env:CMD}/{envname}/zabbix.yaml:{env:COMMON}          -o {env:OUTPUT}
+    zabbix:            {env:CMD}/10.0/zabbix.yaml:{env:COMMON}               -o {env:OUTPUT}
```
Comment (naumvd95) [inline tox.ini]: why u define zabbix as separate env?

File: tox_update.ini [hunk 9]
```diff
@@ -22,5 +22,5 @@ commands =
     9.{x,1,0}:         {env:CMD}/{envname}/vcenter.yaml:{env:COMMON}
     9.{x,1,0}:         {env:CMD}/{envname}/nsxt.yaml:{env:COMMON}
     9.x:               {env:CMD}/{envname}/upgrade.yaml:{env:COMMON}
-    9ui:               {env:CMD}/9.x/ui.yaml:{env:COMMON}
-    10ui:              {env:CMD}/10.0/ui.yaml:{env:COMMON}
\ No newline at end of file
+    9.x,10.0:          {env:CMD}/{envname}/ui.yaml:{env:COMMON}
+    10.0:              {env:CMD}/{envname}/fuel.yaml:{env:COMMON}
```
Comment (naumvd95) [inline tox_update.ini]: where is zabbix
and seems to me, u shud separate this parts on 2 PR

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/10.0/zabbix.yaml (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/10.0/zabbix.yaml:108]: it's ok

File: servers/partner-ci/10.0/zabbix.yaml (hunk 4 above)
Comment (rkhozinov) [reply servers/partner-ci/10.0/zabbix.yaml:355]: ok

File: servers/partner-ci/builders/build-plugin-zabbix.sh (hunk 6 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/build-plugin-zabbix.sh:29]: ok

File: tox.ini (hunk 8 above)
Comment (rkhozinov) [reply tox.ini]: it's useful

File: tox_update.ini (hunk 9 above)
Comment (rkhozinov) [reply tox_update.ini]: it will be added soon

**Pull request closed without merging**